src/vivarium_gates_mncnh/data/lbwsg_paf.yaml

```diff
@@ -32,7 +32,7 @@ configuration:
             day: 2
         step_size: 3 # Days
     population:
-        population_size: 195_112
+        population_size: 122_728

```

Comment:
This takes away the population increase to calculate the preterm PAF but maybe we just need to do two separate calculations

Reply:
I don't think this is a problem, I just wanted to check.